package.json

```diff
@@ -21,11 +22,11 @@
     "prettier": "^1.17.0"
   },
   "dependencies": {
-    "axios": "^0.18.0",
+    "@commercetools/http-user-agent": "^2.0.0",
+    "@octokit/rest": "^16.25.0",
     "dotenv": "^7.0.0",
-    "eslint": "^5.16.0",
     "file-system": "^2.2.2",
     "mri": "^1.1.4",
-    "shelljs": "^0.8.3"
+    "pkg": "^4.3.8"
```

Comment:
🤔

Reply:
Not sure why i've installed shelljs 😅 
But we don't need it 👍

Reply:
No I mean, remove the `pkg`